src/MailTracking.php

```diff
@@ -185,6 +185,26 @@ protected function seeEmailFrom($sender, Swift_Message $message = null)
     }
 
     /**
+     * Assert that the last email had the given priority level.
+     * The value is an integer where 1 is the highest priority and 5 is the lowest.
+     *
+     * @param integer            $priority
+     * @param Swift_Message|null $message
+     *
+     * @return PHPUnit_Framework_TestCase $this
+     */
+    protected function seeEmailPriorityEquals($priority, Swift_Message $message = null)
+    {
+        $actual_priority = $this->getEmail($message)
+                                ->getPriority();
+
+        $this->assertEquals($priority, $actual_priority,
+            "The last email sent had a priority of $actual_priority but expected $priority.");
```

Comment:
@jimmypuckett Nitpick-CI is telling me to format this like the following:

```
$this->assertEquals(
    $priority,
    $actual_priority,
    "The last email sent had a priority of $actual_priority but expected $priority."
);
```

Nothing else has that format at the moment. Should I go refactor the other methods?